src/HttpCommon.jl

```diff
@@ -3,7 +3,6 @@ __precompile__()
 module HttpCommon
 
 using Compat
-using Compat: UTF8String
```

Comment:
Compat isn't needed for that, the new syntax parses on 0.6 but not 0.5 so that deprecation can be addressed when dropping julia 0.5 support